### ISSUE TITLE
Object isn't animating with Empty Object

### ISSUE THREAD

**Issue by RivalC70** [JacquesLucke/animation_nodes#1732]
I'm following this tutorial with animation nodes:

https://www.youtube.com/watch?v=xcjrD5SwQBw

I've followed it up to 4:38 where it explains that the control of the scaling animation is with the Empty object but I've done the exact same (Or at least what looks like the same to me) in the animation node tree but nothing is moving when scaled!

This is my first time with animation nodes so I'm not sure how to scale here.

![image](https://user-images.githubusercontent.com/84719464/119349238-d4b96180-bc95-11eb-83d0-e23f18ee41b5.png)

I'm not sure what I'm doing wrong here, it was going okay so far up until trying to move the empty with the animation.

**Comment by Gerstmann-Bradley** [comment]
![image](https://user-images.githubusercontent.com/55994236/119349740-7b890800-bc6c-11eb-86b6-efe6875fd01d.png)
Have you setup the trigger?

**Comment by RivalC70** [comment]
Linking to map range has changed it but it still doesn't move with the empty - if by triggers, here's what I have at the moment: 

![image](https://user-images.githubusercontent.com/84719464/119351299-3da1d900-bc98-11eb-933c-663621e342dd.png)

**Comment by Gerstmann-Bradley** [comment]
You are using two different empties each for falloff and trigger. switch to the same empty and try

**Comment by RivalC70** [comment]
Fixed, it's updating now and actually working. Thanks for the help and the speed of the reply especially. I didn't think I was going to get one for at least a couple of hours  👍

**Comment by RivalC70** [comment]
One last thing if it's okay - going further into the video, I'm at the part where moving the empty object up and down gives a transition effect from 0-1 then back to 0 again, however it's leaving behind smaller shaped scales. 
This is what it's supposed to be:
![image](https://user-images.githubusercontent.com/84719464/119357644-b6586380-bc9f-11eb-8cfc-aec4e7f5698a.png)

Here's mine: 
![image](https://user-images.githubusercontent.com/84719464/119357734-d0924180-bc9f-11eb-94ef-b72981335cf2.png)

**Comment by Gerstmann-Bradley** [comment]
There are many factors that maybe causing your issue, such the position and scale of your empty, along with many parameters.
Unfortunately It's difficult for me to know what exactly causes your result different from tutorial. I am neither the person who came up with this tutorial.
I guess you have to tweak and investigate by yourself.

**Comment by RivalC70** [comment]
Sorry, the interpolation was connected a second or two after I screenshotted it.

I appreciate the help so far as it's gotten me further into the tutorial! I closed and re-opened it (Without saving) did the fixes you suggested beforehand however the only thing I did was scale the empty down and that's seemed to do it...? I'm not sure why they've disappeared now?

![image](https://user-images.githubusercontent.com/84719464/119361623-c5d9ab80-bca3-11eb-84bc-79132a6bc79a.png)

![image](https://user-images.githubusercontent.com/84719464/119361644-cb36f600-bca3-11eb-87b2-c61e4f36f1c6.png)
